src/font.js

```diff
@@ -87,11 +88,45 @@ function Font(options) {
         this.ascender = options.ascender;
         this.descender = options.descender;
         this.createdTimestamp = options.createdTimestamp;
+        this.italicAngle = options.italicAngle || 0;
+        this.weightClass = options.weightClass || 0;
+
+        let selection = 0;
+        if (options.fsSelection) {
+            selection = options.fsSelection;
+        } else {
+            if (this.italicAngle < 0) {
+                selection |= this.fsSelectionValues.ITALIC;
+            } else if (this.italicAngle > 0) {
+                selection |= this.fsSelectionValues.OBLIQUE;
+            }
+            if (this.weightClass >= 600) {
+                selection |= this.fsSelectionValues.BOLD;
+            }
+            if (selection == 0) {
+                selection = this.fsSelectionValues.REGULAR;
+            }
+        }
+
+        if (!options.panose || !Array.isArray(options.panose)) {
+            options.panose = [0, 0, 0, 0, 0, 0, 0, 0, 0];
+        }
+
         this.tables = Object.assign(options.tables, {
             os2: Object.assign({
                 usWeightClass: options.weightClass || this.usWeightClasses.MEDIUM,
                 usWidthClass: options.widthClass || this.usWidthClasses.MEDIUM,
-                fsSelection: options.fsSelection || this.fsSelectionValues.REGULAR,
+                bFamilyType: options.panose[0] || 0,
+                bSerifStyle: options.panose[1] || 0,
+                bWeight: options.panose[2] || 0,
+                bProportion: options.panose[3] || 0,
```

Comment:
This PR introduces names for the individual Panose values.
I think this is problematic.

When one sees a property `bWeight` one might assume it has something to do with the font weight. But the Panose values are ... a mess. The meaning of a certain value depends on the 1st Panose value.

For example with the 4th Panose value `panose[3]` contains
* `Proportion` for Latin Text
* `Spacing` for Latin Script and Latin Picture
* `Aspect` for Latin Decorative

So I would not call it `bProportion` as that is an oversimplification. I believe it is best to keep just the array and let interested uses decode it by for example this excellent compilation that I usually use for reference:

![image](https://github.com/user-attachments/assets/a402e66a-a2d2-4720-a6af-355e84bb5e35)
_https://forum.high-logic.com/postedfiles/Panose.pdf_